<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Copyright 2016, Christian Kvasny</author>
 <class>ProductEdit</class>
 <widget class="QDialog" name="ProductEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Artikel Editieren</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Artikel Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name">
       <property name="placeholderText">
        <string>Artikelname</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="itemNumLabel">
       <property name="text">
        <string>Artikel Nummer:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="itemNum"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="barcodeLabel">
       <property name="text">
        <string>Barcode:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="barcode"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="categorieLabel">
       <property name="text">
        <string>Gruppe:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="groupComboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Button Farbe:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="colorComboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Steuer:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="taxComboBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="visibleLabel">
       <property name="text">
        <string>Sichtbar:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="visibleCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="collectionReceiptLabel">
       <property name="text">
        <string>Extrabon:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="collectionReceiptCheckBox">
       <property name="text">
        <string>(ExtraBon f√ºr diesen Artikel drucken)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Netto Preis: </string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="net">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="PriceLabel">
       <property name="text">
        <string>Brutto Preis:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="gross"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Lagerstand:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="stockLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Mindestbestand:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="minstockLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QrkPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:src/icons/ok.png</normaloff>:src/icons/ok.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QrkPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Abbrechen</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:src/icons/cancel.png</normaloff>:src/icons/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QrkPushButton</class>
   <extends>QPushButton</extends>
   <header>qrkpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>itemNum</tabstop>
  <tabstop>barcode</tabstop>
  <tabstop>groupComboBox</tabstop>
  <tabstop>colorComboBox</tabstop>
  <tabstop>taxComboBox</tabstop>
  <tabstop>visibleCheckBox</tabstop>
  <tabstop>net</tabstop>
  <tabstop>gross</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
